rfcs/0010-stamps.md

```diff
@@ -0,0 +1,37 @@
+# Stamps: Airdrops with Intent
+
+The goal of this RFC is to define an airdrop mechanism that authorizes funds
+for a particular use.
+
+## Background
+
+To run an on-chain program, a client must first acquire tokens to pay a Solana
+cluster a transaction fee to instantiate the program. To acquire the tokens,
+the client might request them from a special client called a *drone*.
+
+## Problems
+
+1. Once the client has tokens to instantiate the program, the client may choose
+   to simply keep them.
+2. Loading a program requires two transaction fees - one for the airdrop and
```

Comment:
I don't think this is a good example of the problem because:
1) Loading the program will require more than one transaction in most cases (probably dozens to hundreds!)
2) Loading the program is not a client concern.  It just appears this way in our current testnet because the ledger is not permanent.

I think it would be better to reframe this as enabling random joe public to interact with your program, at your cost, without them worrying about what a token is.

Reply:
Is the problem that I used the language "loading a program" instead of "instantiating a program"?

Reply:
Yeah that tweak would definitely help clarify.